.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -896,6 +896,7 @@ npath
 npathcomplexity
 npe
 npm
+ntp
```

Comment:
This can be avoided by using long option name: `--no-transfer-progress`

Reply:
done